pkg/cmd/server/kubernetes/master_config_test.go

```diff
@@ -93,11 +98,13 @@ func TestCMServerDefaults(t *testing.T) {
 			LookupCacheSizeForRC:              4096,
 			LookupCacheSizeForRS:              4096,
 			LookupCacheSizeForDaemonSet:       1024,
+			ConfigureCloudRoutes:              true,
+			NodeCIDRMaskSize:                  24,
 			ServiceSyncPeriod:                 unversioned.Duration{Duration: 5 * time.Minute},
 			NodeSyncPeriod:                    unversioned.Duration{Duration: 10 * time.Second},
 			ResourceQuotaSyncPeriod:           unversioned.Duration{Duration: 5 * time.Minute},
 			NamespaceSyncPeriod:               unversioned.Duration{Duration: 5 * time.Minute},
-			PVClaimBinderSyncPeriod:           unversioned.Duration{Duration: 10 * time.Minute},
+			PVClaimBinderSyncPeriod:           unversioned.Duration{Duration: 15 * time.Second},
```

Comment:
This is a huge change.  Do you know why its so tight now?  If not, we should have an issue to figure out why.  That seems crushing.